include/snappy_boot_v1.h

```diff
@@ -0,0 +1,117 @@
+/**
```

Comment:
this should be in include/lk  , no?

not completely a fan of growing an include dir just for this.  @mvo5 thoughts?

Reply:
@pedronis happy to move it where you like. This was suggestion from @zyga 
I'm myself wondering what's best place to put this. But for reference I do want to have it somewhere in snapd repo

Reply:
moved to `include/lk`